server/routerlicious/packages/services/src/redisSessionManager.ts

```diff
@@ -151,7 +151,7 @@ export class RedisCollaborationSessionManager implements ICollaborationSessionMa
 		return new Promise((resolve, reject) => {
 			const callbackPs: Promise<T>[] = [];
 			sessionJsonScanStream.on("data", (result: string[]) => {
-				if (!result) {
+				if (result.length === 0) {
```

Comment:
This one worries me a bit, given our history of lacking type-safety with events. So I could imagine this one introducing a runtime behavior change. But if that's the case, this still seems worth doing.